tools/ratbagctl.test.in

```diff
@@ -447,10 +479,23 @@ def tearDownModule():
     os.unlink(DBUS_CONF_PATH)
 
 
+def parse(input_string):
+    global run_ratbagctl_in_subprocess
+    parser_test = argparse.ArgumentParser(description="Testsuite for ratbagd/ratbagctl", add_help=False)
+    parser_test.add_argument("--subprocess", action='store_true', default=False)
+    ns, rest = parser_test.parse_known_args(input_string)
+    run_ratbagctl_in_subprocess = ns.subprocess
+    return rest
+
+
 def main(argv):
     if not os.geteuid() == 0:
         sys.exit('Script must be run as root')
 
+    args = parse(sys.argv[1:])
+
+    sys.argv = [sys.argv[0], *args]
+
     unittest.main()
```

Comment:
a call to `unittest.main(argv=args)` is more expressive here, I'm not a fan of overwriting `sys.argv`